enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md

```diff
@@ -164,7 +171,22 @@ const (
 	PublicKeyRootOfTrust         PolicyType = "PublicKey"
 	FulcioCAWithRekorRootOfTrust PolicyType = "FulcioCAWithRekor"
 	PKIRootOfTrust               PolicyType = "PKI"
-)
+
+    // Kuberentes secret data keys for `PublicKey` struct fields
```

Comment:
nit: "Kuberentes" -> "Kubernetes", here and elsewhere, and it's nice to have Godocs that open with the function/variable/struct name, like:

```golang
// SecretPublicKeyDataKey is a Kubernetes Secret data key....
```